src/ripple/app/main/LoadManager.cpp

```diff
@@ -32,38 +33,50 @@
 
 namespace ripple {
 
-class LoadManagerImp
-    : public LoadManager
-    , public beast::Thread
+class LoadManagerImp : public LoadManager
 {
 public:
     //--------------------------------------------------------------------------
 
-    beast::Journal m_journal;
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
-    LockType mLock;
+    beast::Journal mJournal;
+    std::thread mThread;
 
-    bool mArmed;
+    std::mutex mMutex;          // Guards mDeadLock and mArmed
 
     int mDeadLock;              // Detect server deadlocks
+    bool mArmed;
+
     //--------------------------------------------------------------------------
 
     LoadManagerImp (Stoppable& parent, beast::Journal journal)
         : LoadManager (parent)
-        , Thread ("loadmgr")
-        , m_journal (journal)
-        , mArmed (false)
+        , mJournal (journal)
+        , mThread ()
+        , mMutex ()
```

Comment:
I'm fine with that change.  FWIW, the reason I include the default constructors in the initializer list is that it makes it very easy to audit how each member is initialized.  And it adds no runtime cost.  Bu tI don't feel strongly about it.